mpdev/cmd/commands.go

```diff
@@ -34,6 +36,17 @@ func GetMpdevCommands(name string) (c []*cobra.Command) {
 	return c
 }
 
+// Replace occurrences of "kpt" with "mpdev" in docs.
+func fixDocs(old string, new string, cmd ...*cobra.Command) {
+	for _, c := range cmd {
+		c.Use = strings.ReplaceAll(c.Use, old, new)
```

Comment:
Is there anyway replace whole words, to avoid potential partial matches? I guess we can also craft a regex like `\wkpt\w`

Reply:
Done